lib/models/movie.js

```diff
@@ -4,7 +4,7 @@ const Bookshelf = require('../libraries/bookshelf');
 
 module.exports = Bookshelf.Model.extend({
   tableName: 'movies',
-  serialize: () => {
+  serialize: function() {
```

Comment:
Converting it back to function() instead of callback, since it throws an error (`this.get is not a function`) during mocha tests. Maybe callback is not supported by it.

Reply:
Yeah! I actually learned about this recently too, cuz according to the Javascript spec:

> An arrow function expression is a syntactically compact alternative to a regular function expression, although without its own bindings to the this, arguments, super, or new.target keywords. Arrow function expressions are ill suited as methods, and they cannot be used as constructors.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

Reply:
Interesting, I wonder what `this` is being bound to in the arrow function.

Reply:
Oh arrow functions can't be used as methods